test/centered_continued_fraction_test.cpp

```diff
@@ -8,6 +8,7 @@
 #include "math_unit_test.hpp"
 #include <boost/math/tools/centered_continued_fraction.hpp>
 #include <boost/math/constants/constants.hpp>
+#include <boost/core/demangle.hpp>
```

Comment:
I have added a demangler to `math_unit_test.hpp` to facilitate standalone; it's available via `detail::demangle`.